### DISCUSSION TITLE
Looking for a community

### DISCUSSION THREAD

**Discussion by Ifra001** [Cyfrin/foundry-full-course-cu#2777] [General]
Is there a discord community or any online community where people discuss their progress in the blockchain space, provide their experiences and tips and stuff. Tbh, when you are in a community about a specific field, you feel more connected and motivated to be consistent in it every day.

**Comment by EngrPips** [comment]
Are you in Cyfrin Discord @DontMind-me?

**Reply by wenahyuck** [reply]
Can you send the invite for their discord?

**Reply by EngrPips** [reply]
Yeah sure https://discord.gg/T8wJ6fMT

**Reply by Ifra001** [reply]
Yupp, I'm there!

**Reply by EngrPips** [reply]
Awesome. It is a great and big community where you will find like-minded minds.

**Reply by JamesLivengood** [reply]
> Yeah sure https://discord.gg/T8wJ6fMT

Could you send an updated link? I can't find the link on the Cyfrin Updraft website

**Reply by EngrPips** [reply]
Here https://discord.gg/e7Qds8mw
